README.md

```diff
@@ -120,12 +141,27 @@ Then set the GDB Server to be JLinkGDBServer, ie:
 ```
 C:\Program Files (x86)\SEGGER\JLink\JLinkGDBServer.exe
 ```
+
+##### Mac OS
+Under **File->Settings->Build, Execution, Deployment...->Toolchains**:
+Set the default C compiler to be the GNU C and C++ compilers you installed earlier. If you installed gcc using brew it should be something similar:
```

Comment:
What do you mean here `If you installed gcc using brew it should be something similar`?

Reply:
I meant that the C/C++ compiler paths should be similar to the ones I list below. Updated readme to make this more explicit